configs/setup_networking.py

```diff
@@ -257,9 +257,15 @@ def __init__(self, manconfig):
 
         for dest in (self.tftp_grub_efi, self.sdhc_grub_efi):
             base = os.path.basename(dest)
-            grubURL = manconfig['L4TM_MIRROR'] + \
-                '/dists/catapult/main/uefi/grub2-arm64/' +\
-                'current/%s.signed' % base
+            # Override grubURL by manconfig value, if it is set.
+            # Otherwise, pull from mirror/dists/.../uefi/
+            grubURL = manconfig.get('GRUB_EFI_BASE_URI', None)
+            if grubURL:
+                grubURL += base
+            else:
+                grubURL = manconfig['DEBIAN_MIRROR'] + \
```

Comment:
These do not exist on any stock Debian mirror.  The default should be hardcoded to

http://rocky42.americas.hpqcorp.net/MFT/grub

Until I get it into somewhere better.

Reply:
Line 266 here is temporary to not to break this routine. We well be using "GRUB_EFI_BASE_URI" value from tmms. Line 266 will go away soon, but keep it for this merge, just in case.

Reply:
0) We're trying to get away from the L4TM repo.  This goes right back to it.
1) It's now called DEBIAN_MIRROR, it's not on seedy, so this is broken.
2) It's on my server for now.  That's good enough to move forward (internally).

Reply:
Alright. I will just remove 266 line. Use tmms config to get your grub.

Reply:
That's good.